modules/setting-up-cpu-manager.adoc

```diff
@@ -101,6 +99,13 @@ cpuManagerReconcilePeriod: 5s   <2>
 <1> `cpuManagerPolicy` is defined when you create the `KubeletConfig` CR.
 <2> `cpuManagerReconcilePeriod` is defined when you create the `KubeletConfig` CR.
 
+. Create a project called `cpuman` by running the following command:
++
+[source,terminal]
+----
+$ oc new-project cpuman
```

Comment:
Contextually, this reads as you must use the cpuman project. Perhaps saying for example?

Reply:
Yes, true. 
Silke we need to change here and add cpuman as an example or something.